diff --git a/CMakeLists.txt b/CMakeLists.txt
index 015cbddee3..659f423ee5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,8 +58,7 @@ if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Emscripten")
     string(APPEND CADICAL_CXXFLAGS " -DNCLOSEFROM")
     ExternalProject_add(cadical
       PREFIX cadical
-      GIT_REPOSITORY https://github.com/arminbiere/cadical
-      GIT_TAG rel-2.1.2
+      URL ./cadical
       CONFIGURE_COMMAND ""
       BUILD_COMMAND $(MAKE) -f ${CMAKE_SOURCE_DIR}/src/cadical.mk
         CMAKE_EXECUTABLE_SUFFIX=${CMAKE_EXECUTABLE_SUFFIX}
@@ -77,8 +76,7 @@ endif()
 if (USE_MIMALLOC)
   ExternalProject_add(mimalloc
     PREFIX mimalloc
-    GIT_REPOSITORY https://github.com/microsoft/mimalloc
-    GIT_TAG v2.2.3
+    URL ./mimalloc
     # just download, we compile it as part of each stage as it is small
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
